sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubProducerAsyncClient.java

```diff
@@ -44,9 +44,11 @@
 import java.util.stream.Collector;
```

Comment:
Did you add a unit test to ensure this AZ_NAMESPACE_KEY is set when sending an event?

Reply:
updated existing tests to assert if AZ_NAMESPACE_KEY set on sending an event.